src/components/App/App.js

```diff
@@ -1,80 +1,34 @@
-import React, {useState, useEffect} from 'react';
-import ItemList from '../ItemList/ItemList';
-import InputItem from '../InputItem/InputItem';
-import Footer from '../Footer/Footer';
-import styles from './App.module.css';
-
-
-const App = () => {
-	const initialState = {
-		items: [
-			{
-				value: 'Покормить кота',
-				isDone: true,
-				id: 1
-			},
-			{
-				value: 'Сходить на работу',
-				isDone: false,
-				id: 2
-
-			},
-			{
-				value: 'Полить цветы',
-				isDone: true,
-				id: 3
-			}
-		],
-
-		count: 3,
-	};
-
-const [items, setItems] = useState(initialState.items);
-const [count, setCount] = useState(initialState.count);
-
-useEffect(() => {console.log('componentDidMount')}, []);
-useEffect(() => {console.log('componentDidUpdate')});	
+import React from 'react';
+import {
+    BrowserRouter as Router,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 1 tab.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)